ecosystem/sep-0038.md

```diff
@@ -365,66 +375,25 @@ Name | Type | Description
 `id` | string | The `id` specified in the request.
 `expires_at` | [UTC ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) string | The date and time by which the anchor must receive funds from the client.
 `price` | string | The price offered by the anchor for one unit of `buy_asset` in terms of `sell_asset`. In traditional finance, `buy_asset` would be referred to as the base asset and `sell_asset` as the counter asset.
-`buy_amount` | string | The amount of `buy_asset` to be exchanged for `sell_asset`. `price * buy_amount = sell_amount` must be true up to the number of decimals required for `buy_asset`.
+`sell_asset` | string | The asset the client would like to sell. Ex. `USDC:G...`, `iso4217:ARS`
 `sell_amount` | string | The amount of `sell_asset` to be exchanged for `buy_asset`.
+`buy_asset` | string | The asset the client would like to exchange for `sell_asset`.
+`buy_amount` | string | The amount of `buy_asset` to be exchanged for `sell_asset`. `price * buy_amount = sell_amount` must be true up to the number of decimals required for `buy_asset`.
 
-##### Examples
```

Comment:
Can we keep the examples deleted examples but just move it to the `GET /price` endpoint?

Reply:
There are already plenty of examples for the `GET /price` endpoint [here](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0038.md#examples-1).

The examples I removed were using `GET /price` requests but showing responses with fields from `GET /quote`, so they wouldn't fit any of those endpoints.